### PR TITLE
Debian packaging

### PR DESCRIPTION
**Pull request by DmitryZabolotsky**
Debian package includes:
 - user creation;
 - directories and permissions;
 - unit file for systemd

Signed-off-by: Dmitriy Zabolotskiy <d.zabolotskiy@yadro.com>

### CODE REVIEW

File: debian/neofs-s3-gw.install [hunk 1]
```diff
@@ -0,0 +1,3 @@
+config/config.yaml etc/neofs/s3
+config/rules.json var/lib/neofs/s3
+bin/neofs-s3-gw usr/bin
```
Comment (alexvanin) [inline debian/neofs-s3-gw.install:3]: What about `bin/neofs-s3-authmate`? @realloc

File: debian/neofs-s3-gw.postinst [hunk 2]
```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+# postinst script for neo-go
```
Comment (alexvanin) [inline debian/neofs-s3-gw.postinst:2]:
```suggestion
# postinst script for neofs-s3-gw
```

File: debian/neofs-s3-gw.postrm [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+# postrm script for neo-go
```
Comment (alexvanin) [inline debian/neofs-s3-gw.postrm:2]:
```suggestion
# postrm script for neofs-s3-gw
```

File: debian/neofs-s3-gw.preinst [hunk 4]
```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+# preinst script for neo-go
```
Comment (alexvanin) [inline debian/neofs-s3-gw.preinst:2]:
```suggestion
# preinst script for neofs-s3-gw
```

File: debian/neofs-s3-gw.prerm [hunk 5]
```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+# prerm script for neo-go
```
Comment (alexvanin) [inline debian/neofs-s3-gw.prerm:2]: neofs-s3-gw

File: debian/neofs-s3-gw.service [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+ExecStart=/usr/bin/neofs-s3-gw --config /etc/neofs/s3/config.yaml
+User=neofs-s3
+Group=neofs-s3
+WorkingDirectory=/var/lib/neofs/s3
+Restart=always
+RestartSec=5
+PrivateTmp=true
+
+[Install]
+WantedBy=multi-user.target
```
Comment (alexvanin) [inline debian/neofs-s3-gw.service:16]: New line in the end.

File: debian/neofs-s3-gw.install (hunk 1 above)
Comment (DmitryZabolotsky) [reply debian/neofs-s3-gw.install:3]: My fault. Fixed.

**Review by alexvanin — APPROVED** [review]

**Review by alexvanin — APPROVED** [review]

**Pull request merged**